File: DDS.Core/Entities/Game.cs
using System.Collections.Generic;
using System.Text.Json.Serialization;

#nullable disable

namespace DDS.Core.Entities
{
    public partial class Game
    {
        public Game()
        {
            Carts = new HashSet<Cart>();
            GameGenres = new HashSet<GameGenre>();
            Ownerships = new HashSet<Ownership>();
        }

        public int GameId { get; set; }
        public string Developer { get; set; }
        public decimal Price { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public decimal Rating { get; set; }
        public int ReviewCount { get; set; }
        public byte[] Image { get; set; }
        public bool IsRemoved { get; set; }

        public virtual ICollection<Cart> Carts { get; set; }
        public virtual ICollection<GameGenre> GameGenres { get; set; }
        [JsonIgnore]
        public virtual ICollection<Ownership> Ownerships { get; set; }
    }
}

File: DDS.Client/ClientApp/src/app/models/game.ts
import { GameGenre } from "./gameGenre";

export class Game {
  get genresString(): string {
    var result = this.gameGenres.map(a => a.genre.name).join();
    console.log(result);
    return result;
  }

  imageBase64: string;

  constructor(
    public gameId?: number,
    public name?: string,
    public developer?: string,
    public description?: string,
    public rating?: number,
    public reviewCount?: number,
    public price?: number,
    public gameGenres?: GameGenre[],
    public image?: ArrayBuffer,
  ) {
    this.imageBase64 = btoa(String.fromCharCode(...new Uint8Array(image)))
    console.log(this.imageBase64);
  }
}
